projects/igniteui-angular/src/lib/date-picker/date-picker.component.spec.ts

```diff
@@ -1235,6 +1236,26 @@ describe('IgxDatePicker', () => {
             expect(datePicker.onDisabledDate.emit).toHaveBeenCalledTimes(2);
         }));
 
+        it('should ignore initial input event from IE', fakeAsync(() => {
+            // #8165
+            // IE fires an input event when the page loads
+            // This causes the DatePicker to gain focus and prevents
+            // the user from blurring the editable input
+            spyOn({ isIE: isIE }, 'isIE').and.returnValue(true);
```

Comment:
Unfortunately, this doesn't do anything for the actual `isIE` function in the utils module though. I see now it's on of those direct exports, which in ES6 modules are readonly and so you can't spy on it.

Since the underlying issue is mostly on the mask, it would've been nice to have a test for it, but this might not be easy to automate after all - there's one more IE quirk here- `setSelectionRange` moves focus on IE, but not on other browsers (apparently it used to on Chrome, but not anymore).